{{cookiecutter.folder_name}}/Makefile

```diff
@@ -36,12 +37,17 @@ tabdir=./tables
 tables=$(tabdir)/*.tex
 styles=./texmf/styles/*.{sty,cls}
 bibs=./texmf/bib/*.bst
-source=$(main).{tex,bbl,bib} lsstdesc.bib macros.tex authors.tex acknowledgments.tex
+source=$(main).{tex,bbl,bib} lsstdesc.bib authors.tex acknowledgments.tex
```

Comment:
Good point. At the moment it is updateable but is not in the texmf tree. I worry that authors will never look in there, and so won't know that they have all these LSST DESC publications to cite. Maybe it'll just become well known? And there's always the README...